ui/src/views/WorkerManager/WMEditWorkerPool/index.jsx

```diff
@@ -64,171 +34,74 @@ export default class WMEditWorkerPool extends Component {
 
   state = {
     workerPool: {
-      name: '',
+      workerPoolId1: '',
+      workerPoolId2: '',
       description: '',
       owner: '',
-      wantsEmail: false,
-      providerType: providers.GCP,
-      providerId: '',
-      config: gcpConfig,
+      emailOnError: false,
+      config: PROVIDER_CONFIGS.get(GCP),
     },
-    invalidProviderConfig: false,
   };
```

Comment:
The state doesn't seem to change so I think we can remove this from here. This seems more the job of the editor component.

Reply:
but it's removed o_O (you commented on deletion)

Reply:
done